main.py

```diff
@@ -36,7 +36,7 @@
 )
 # Modified by JasonLovesDoggo
 print_markdown(
-    "### Thanks for using this tool! [Feel free to contribute to this project on GitHub!](https://lewismenelaws.com) If you have any questions, feel free to reach out to me on Twitter or submit a GitHub issue. You can find solutions to many common problems in the [Documentation](https://luka-hietala.gitbook.io/documentation-for-the-reddit-bot/)"
+    "### Thanks for using this tool! [Feel free to contribute to this project on GitHub!](https://lewismenelaws.com) If you have any questions, feel free to reach out to me on Twitter or submit a GitHub issue. You can find solutions to many common problems in the [Documentation]: https://reddit-video-maker-bot.netlify.app/"
 )
```

Comment:
It's markdown You can't add a space and a colon

Reply:
I know but it was hidden in the code because you can't click it or something. So I thought that it would be better to just show it. Do I just need to replace the URL so it is hidden or show it?

Reply:
Done!

Reply:
> Just replace it exactly how it was before but with a new URI Don't try to change the markdown

The way Simon implemented is better since markdown links don't work in terminals - might as well have just not put it in otherwise

Reply:
Okay, I reverted it! I left the () in the code to make the "Documentation" text blue.